src/App.tsx

```diff
@@ -1,14 +1,74 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { InitialStates, Todo } from './types/Todo';
+import { getTodos, getUser } from './api';
+import { User } from './types/User';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [toggleModal, setToggleModal] = useState<boolean>(false);
+  const [selectedTodo, setSelectedTodo] = useState<Todo>();
+  const [user, setUser] = useState<User>();
+  const [filterTodos, setFilterTodos] = useState<InitialStates>({
```

Comment:
`filterTodos` sounds like the method name

Please rename (`todosFilter`, `filter`, etc)